### PR TITLE
Updates from email feedback

### PR DESCRIPTION
**Pull request by brianmajor**

### CODE REVIEW

File: GMS.tex [hunk 1]
```diff
@@ -47,7 +47,7 @@ \section{Introduction}
...
 
-The ideas presented by GMS enable data centres to do authorization checks in an interoperable fashion.  In the context of authorization, interoperability can viewed on two levels:  interoperability amongst the cooperating services \emph{within} a data centre, and interoperability \emph{between} data centres.  Because of the orthogonal nature of authorization, these amount to the same problem.
+The ideas presented by GMS enable data centres to do authorization checks in an interoperable fashion.  In the context of authorization, interoperability can viewed on two levels:  interoperability amongst the cooperating services \emph{within} a data centre, and interoperability \emph{between} data centres.  The same approach is taken for both scenarios, meaning that the location of a GMS service is irrelavent.
```
Comment (msdemlei) [inline GMS.tex]: Typo: irrelevant

**Review by msdemlei — DISMISSED** [review]
Other than a typo I've annotated, I'm fine with this.  Perhaps one of the Asterics-funded persons could comment whether they're ok with the "was" for ASTERICS.

File: GMS.tex [hunk 2]
```diff
@@ -26,7 +26,7 @@
 \end{abstract}
 
 \section*{Acknowledgments}
-For the creation of this document we acknowledge the support of the Canadian Space Agency, the National Research Council Canada, the Italian National Institue of Astrophysics, and the Astronomy ESFRI and Research Infrastructure Cluster (ASTERICS).  ASTERICS is a project supported by the European Commission Framework Programme Horizon 2020 Research and Innovation action under grant agreement n. 653477.
+For the creation of this document we acknowledge the support of the Canadian Space Agency, the National Research Council Canada, the Italian National Institue of Astrophysics, and the Astronomy ESFRI and Research Infrastructure Cluster (ASTERICS).  ASTERICS was a project supported by the European Commission Framework Programme Horizon 2020 Research and Innovation action under grant agreement n. 653477.
```
Comment (molinaro-m) [inline GMS.tex]: what about simply:
"...Cluster (ASTERICS, European Commission Horizon 2020 RIA grant agreement n. 653477)."
It eliminates the need for "is/was".

**Comment by brianmajor** [comment]
@gtaffoni @pdowler  Who should be approving and merging pull requests for the GMS RFC?  I have a number of updates I'd like to incorporate.  Should I be using different branches to keep the PRs separate?

**Review by gtaffoni — APPROVED** [review]
approved

**Pull request merged**